frontend/src/component/admin/network/NetworkTrafficUsage/BackendConnections.tsx

```diff
@@ -0,0 +1,75 @@
+import type { FC } from 'react';
+import { usePageTitle } from 'hooks/usePageTitle';
+import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { Alert, Box } from '@mui/material';
+import { PeriodSelector } from './PeriodSelector';
+import { Bar } from 'react-chartjs-2';
+import { customHighlightPlugin } from 'component/common/Chart/customHighlightPlugin';
+import { getChartLabel } from './chart-functions';
+import { useConsumptionStats } from './hooks/useStats';
+import { StyledBox, TopRow } from './SharedComponents';
+import {
+    BarElement,
+    CategoryScale,
+    Chart as ChartJS,
+    Legend,
+    LinearScale,
+    Title,
+    Tooltip,
+} from 'chart.js';
+import annotationPlugin from 'chartjs-plugin-annotation';
+import { useChartDataSelection } from './hooks/useChartDataSelection';
+
+export const BackendConnections: FC = () => {
```

Comment:
Similar to NetworkTrafficUsage but fewer items (removed warnings about overages since they do't apply here)